### ISSUE TITLE
CREATE TEMPORARY TABLE is not supported

### ISSUE THREAD

**Issue by marcus-sa** [tursodatabase/libsql#441]
`SQL string could not be parsed: unsupported statement: CREATE TEMPORARY TABLE _tmp_d as SELECT "id" FROM "user" WHERE "user"."name" = ? LIMIT 1;`

**Comment by psarna** [comment]
Temporary tables and views work fine in libSQL (the local embedded db engine), but they're not supported in sqld (the server), because their lifetime is bound to a single database connection, and it gets more complicated with a primary-replica architecture, where users can read from any replica server, but always write to a single primary. There are ways around that and one day we may figure out a way to allow users to create TEMP tables and views, but it's not supported in sqld at the moment.

**Comment by marcus-sa** [comment]
@psarna fair enough. I can't add support for Turso in https://github.com/deepkit/deepkit-framework then.
